src/main/java/edu/berkeley/cs186/database/query/SortOperator.java

```diff
@@ -86,8 +86,15 @@ public Iterator<Record> iterator() {
      * iterator
      */
     public Run sortRun(Iterator<Record> records) {
-        // TODO(proj3_part1): implement
-        return null;
+        // TODO(proj3_part1): implement Done
+
+        List<Record> recordList = new ArrayList<>();
+        while (records.hasNext()) recordList.add(records.next());
+        recordList.sort(new RecordComparator());
+
+        Run sortedRun = new Run(transaction , getSchema());
+        sortedRun.addAll(recordList);
+        return sortedRun;
```

Comment:
```suggestion
        return makeRun(recordList);
```
맨 아래에 recordList를 추가하여 새로운 Run 객체를 생성해주는 메소드가 구현되어 있습니다! 활용해보시는 것도 좋을 것 같아요.